tla/ccfraft.tla

```diff
@@ -1284,6 +1277,8 @@ MonotonicCommitIndexProp ==
     [][\A i \in Servers :
         commitIndex[i]' >= commitIndex[i]]_vars
```

Comment:
Add a note that `CommittedLogNeverChangesProp` is a much stronger variant of `MonotonicCommitIndexProp`?